Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
OK, I've confirmed that Rails `to_param` works with Globalize and Adminstrate. As mentioned previously, I like Rails `to_param` as it's extremely light weight.

https://github.com/montrealrb/Montreal.rb/commit/21dfe9633a2dbc9da4b2eae91496e4a12dc9b2e2

But @sophiedeziel has the last call on the stack so I will defer to her. I'll shut up now :smile_cat: